doc/source/creating.rst

```diff
@@ -37,7 +37,7 @@ The next step is to install a version sentinel. This is done as follows:
     db_analysis = Broker.named(config_name)
 
     # install sentinels
-    databroker.assets.utils.install_sentinels(db_analysis.reg.config,
+    db_analysis.assets.utils.install_sentinels(db_analysis.reg.config,
```

Comment:
I think the first commit was just right. Also, when Julien created this piece of documentation, he later mentioned that it should be `db` instead of `db_analysis`:
```py
     db = Broker.named(config_name)
     ....
     databroker.assets.utils.install_sentinels(db.reg.config, version=1)
```

Reply:
Ah OK, thanks @mrakitin for the clarification, I was able to get it to work in both ways. I am happy with either solution.